modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -134,10 +134,12 @@
                         "properties": {
                           "id": {
                             "type": "string",
-                            "description": "Claim ID from EVSS"
+                            "description": "Claim ID from EVSS",
+                            "example": "8347210"
```

Comment:
why was this v1 swagger doc updated with a bunch of example values?

Reply:
[Because I updated the schema file to include the example values that already exist today.](https://github.com/department-of-veterans-affairs/vets-api/pull/7195/files#r649196331)